indico/modules/rb/__init__.py

```diff
@@ -69,13 +71,13 @@ def _extend_admin_menu(sender, **kwargs):
 
 @signals.menu.items.connect_via('top-menu')
 def _topmenu_items(sender, **kwargs):
-    if config.ENABLE_ROOMBOOKING:
+    if config.ENABLE_ROOMBOOKING and rb_check_if_visible(session.user):
         yield TopMenuItem('room_booking', _('Room booking'), url_for('rb.roombooking'), 80)
 
 
 @signals.menu.items.connect_via('event-management-sidemenu')
 def _sidemenu_items(sender, event, **kwargs):
-    if config.ENABLE_ROOMBOOKING and event.can_manage(session.user):
+    if config.ENABLE_ROOMBOOKING and rb_check_if_visible(session.user) and event.can_manage(session.user):
```

Comment:
This one is no longer needed

Reply:
this page already has a check access and gives Forbidden(_('You are not authorized to access the room booking system.')) for unauthorised users so I think it's okay to hide that link as well since the UI is already unaccessible. Unless you definitely want to keep that link

Reply:
ok, then maybe that access check is too strict... anyway, in that case you can ignore my previous comment about just hiding the UI on the page while still showing the buttons - or relax the access check for `RHEventBookingList`.

Reply:
I relaxed the check_access for that page. So the UI is now visible but not the buttons based on the settings. 

BTW I'm hiding the entire row to book if the users shouldn't see the rb because just removing the buttons doesn't make it make sense.

## Visible: 
<img width="826" alt="Screenshot 2023-10-09 at 13 03 45" src="https://github.com/indico/indico/assets/54508387/af568e2c-4178-4fee-92d4-ac7121afda94">

## Hidden:
<img width="834" alt="Screenshot 2023-10-09 at 13 04 00" src="https://github.com/indico/indico/assets/54508387/0012a634-3deb-4f62-beb1-5571b3155dcf">

And If the event has no booking: 
## Visible:
<img width="852" alt="Screenshot 2023-10-09 at 13 06 03" src="https://github.com/indico/indico/assets/54508387/67ff3fbc-d954-40c4-9f30-9d8675cc1b9a">

## Hidden: 
<img width="681" alt="Screenshot 2023-10-09 at 13 05 49" src="https://github.com/indico/indico/assets/54508387/410b0441-a0cc-41c1-8e06-d95757c33d7f">